data-prepper-plugins/otel-trace-raw-processor/README.md

```diff
@@ -12,9 +12,14 @@ processor:
 ## Configuration
 
 * `trace_flush_interval`: An `int` represents the time interval in seconds to flush all the descendant spans without any root span. Default to 180.
+* `trace_group_cache_ttl`: A `Duration` represents the time-to-live for traces in the trace group cache. Defaults to 15 seconds.
+* `trace_group_cache_max_size`: An `int` representing the total number of traces to keep in the trace group cache.
 
 ## Metrics
-Apart from common metrics in [AbstractProcessor](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-api/src/main/java/com/amazon/dataprepper/model/processor/AbstractProcessor.java), `otel_trace_raw` processor has not introduced any custom metrics.
+In addition to the metrics from [AbstractProcessor](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-api/src/main/java/com/amazon/dataprepper/model/processor/AbstractProcessor.java):
```

Comment:
[AbstractProcessor](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-api/src/main/java/com/amazon/dataprepper/model/processor/AbstractProcessor.java) link is now 404. Looks like there's no README in that sub-directory [level](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-api/src/main/java/com/amazon/dataprepper/model/processor/AbstractProcessor.java) . We'd want to add the common Processor metrics there to link them here.

Reply:
Nice catch. I'll address.

Reply:
That was a good catch. It inspired me to fix other links, so I created a new PR for other ones: #2186

Reply:
Thanks for fixing the other ones too!